builder/digitalocean/config.go

```diff
@@ -187,6 +195,20 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 		}
 	}
 
+	// Check if the PrivateNetworking is enabled by user before use VPC UUID
```

Comment:
I think it makes sense to add a few tests to builder/digitalocean/builder_test.go to make sure this template validation logic works as expected.